### DISCUSSION TITLE
LiquidMenu with I2C 16x2 display

### DISCUSSION THREAD

**Discussion by DeepeshPatidar** [VasilKalchev/LiquidMenu#111] [Q&A] [answered]
I am using 16x2 LCD display with I2C module on it. I am using LiquidCrystal_I2C library to operate LCD using 2 wires I2C communication while the LiquidMenu library using LCD with 4 data wires connection. How can I use I2C display with LiquidMenu library?

**Comment by VasilKalchev** [comment, marked as answer]
Duplicate of #114.
